proto/storage/compliance.proto

```diff
@@ -60,17 +57,41 @@ message ComplianceResultValue {
 }
 
 message ComplianceControlResult {
-    ComplianceResource resource = 1;
-    string control_id           = 2;
-    ComplianceResultValue value = 3;
+    ComplianceResource    resource   = 1;
+    string                control_id = 2;
+    ComplianceResultValue value      = 3;
 }
 
 // Next available tag: 5
 message ComplianceDomain {
     string id = 4 [(gogoproto.moretags) = 'sql:"pk" search:"Compliance Domain ID,hidden"'];
 
+    // These must mirror the tags _exactly_ in cluster.proto for backwards compatibility
+    message Cluster {
+        string id   = 1;
+        string name = 2;
```

Comment:
I think it might be nice to add `reserved 3 to <max fieldnum in cluster proto>` to prevent any accidents, but don't feel too strongly, because it'll be hard to keep this in sync with the actual `Cluster` proto.